docs/current_docs/manuals/developer/dependencies.mdx

```diff
@@ -77,6 +77,10 @@ You can also use local modules as dependencies. However, they must be stored in
 dagger install ./path/to/module
 ```
 
+:::note
+Installing a local module using a relative path is only possible if your project is initialized as a Git repository (the project root has a `.git` directory).
```

Comment:
It's not just relative paths, this is also true for absolute paths. It's rather about using local paths vs git remote paths. 

For **local paths**, the module you want to install needs to be **within** the same context directory, i.e., repo root if in a git repo, or root directory otherwise (directory where `dagger.json` is).

Reply:
Updated, see if better?